scripts/peer-dependency-generator/index.js

```diff
@@ -0,0 +1,43 @@
+#!/usr/bin/env node
+/* eslint-disable no-console */
+/* eslint import/no-extraneous-dependencies: ["error", {"devDependencies": true}] */
+const fs = require('fs');
+const path = require('path');
+const packagePaths = require('./getPackagePaths');
+const generatePeerDependenciesDoc = require('./generatePeerDependencyDoc');
+const findAndReplace = require('./findAndReplace');
+
+// Generate the markdown to display peerDependencies information for each package
+packagePaths.forEach((packagePath) => {
+  const packageFile = path.resolve(packagePath, 'package.json');
+  const readmeFile = path.resolve(packagePath, 'README.md');
+  const docReadmeFile = path.resolve(packagePath, 'docs/README.md');
+
+  if (!fs.existsSync(packageFile) || !fs.existsSync(readmeFile) || !fs.existsSync(docReadmeFile)) {
+    return; /* eslint-disable-line no-useless-return */
+  }
+
+  // Read package.json and pull out peerDependencies
+  fs.readFile(packageFile, 'utf8', (err, packageJson) => {
+    if (err) {
+      console.error(`Error reading file ${packageFile} ${err}\n`);
+    } else {
+      const { peerDependencies } = JSON.parse(packageJson);
+
+      if (peerDependencies) {
+        const peerDependenciesDoc = generatePeerDependenciesDoc(peerDependencies);
+        const regex = /<!--.AUTO-GENERATED-CONTENT.START.Peer.Dependencies.-->(.|\n)*<!--.AUTO-GENERATED-CONTENT:END.-->/g;
```

Comment:
Do you manually add the ```<!--.AUTO-GENERATED-CONTENT.START.Peer.Dependencies.-->``` tag to the readme?

Reply:
Yeah, you have to add this as a place holder the first time. After that, we will write it to the file whenever we update the contents.